app/models/school.rb

```diff
@@ -73,6 +74,10 @@ def is_further_education?
     type == 'FurtherEducationSchool'
   end
 
+  def is_la_funded_place?
```

Comment:
As mentioned on the call, if we use the more standard Ruby method predicate method name `la_funded_place?` then in our specs we can use `expect(x).to be_an_la_funded_place`